packages/sdk-ui-ts/src/utils/mappings.ts

```diff
@@ -311,4 +311,6 @@ export const validatorAddressToPathMap = {
     'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp',
     injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj:
     'injvaloper175x4lvxpwmmwt5m76yqz0ace8dwkjyjjj7sxdj.webp'
+  injvalcons1n5yluh6k9zqv74jw35r5s5zy808xpzfqp2wm7g:
```

Comment:
Missing wrapping the address in `'` here and you also miss the `injvaloper...` address mapping

Reply:
updated